_posts/2019-09-09-what-is-mtls.md

```diff
@@ -4,25 +4,63 @@ current: post
+mTLS is just a type of TLS (Transport Layer Security). This is commonly found in verifying endpoints and for encrypting communications and also browsers. The 'https' in a url is a sign of a TLS controlled website. This part of the url verifies the identity of the server, the website you're accessing, to you, the client. 
+Unlike a password which is manually entered by a single user, TLS is typically managed by cryptographic certificates. Most TLS protocols usually verify this with an x509 certificate; and it's typically the client requesting a valid certificate from the server (DNS name on server must match that of the certificate). Secure Sockets Layer (SSL) security is based off this principle and https is its successor.
 
-## How can this affect you and your company?
-For companies, the financial implications of security breaches are tremendous and difficult to recover from. It is not only the brand’s reputation plummeting but it is also the loss of customers and most importantly the huge fines incurred. The brand’s reliability is questioned and trust is lost.
+The main thing that makes mTLS different (and arguably more secure) is that it requires *both* the server and client to verify each other: a handshake. This is particularly useful in instances where you want two servers to communicate with each other; or in a scenario where *you* are the server, not the client. Ultimately it's flexible and lightweight being perfect for edge devices.
 
-## Takeaway
-Machine identity is a critical element of a robust security system for IoT.  Issuing x509 certificates to devices (and rotating them) provides cryptographically verifiable identities that can be used in mutual authentication through mTLS. This limits the chances of adversaries compromising your devices and keeps their hands off your credentials or confidential data. mTLS is a powerful way to enhance security posture, and ensure that your company is taking the necessary precautions to prevent a breach.
+This might still seem a bit overcomplicated. At WoTT we have our own dedicated Certificate Authority and examples of using the CA to secure mTLS connections. We encourage you to try it out to familiarise yourself with the concept. Additionally, our agent is lightweight and can be used flexibly on servers and other edge devices.  
+
+## <a name = "tutorials"> </a> How to implement mTLS: some tutorials by WoTT
```

Comment:
This is broken code. needs to be `<a href="tutorials">...`. However, you can remove it entirely, as Jekyll automatically generates an `id` tag for each header, which you can use as an anchor link.